clang/test/Preprocessor/init.c

```diff
@@ -374,8 +374,7 @@
 // CHERI64:           #define __UINTPTR_MAX__ 4294967295U
 // CHERI64-NEXT:      #define __UINTPTR_TYPE__ unsigned int
 // CHERI64-NEXT:      #define __UINTPTR_WIDTH__ 32
-// CHERI-COMMON:      #define __capability __attribute__((cheri_capability))
```

Comment:
I like that it's no longer a macro since it means you can no longer use `#ifdef __capability` instead of a proper check.

Reply:
:)
(I did consider making it define __capability to __capability like some C headers will do for things then promptly decided that was a bad idea :D)